lib/psulib_traject/indexer.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+require 'traject'
+require 'sidekiq'
+require 'sidekiq-scheduler'
+require 'redis'
+
+class HourliesWorker
+  include Sidekiq::Worker
```

Comment:
These should be under the PsulibTraject namespace, and each worker in its own file. You could define a base work class to include your worker. This like what Rails does.

``` ruby
module PsulibTraject
  class Worker
      include Sidekiq::Worker
  end
end
```

then,

``` ruby
module PsulibTraject
  class HourliesWorker < Work
  end
end
```
and so on....

Alternatively, there's `PsulibTraject::Workers::Base` and `PsulibTraject::Workers::Hourly`

Everything should be required in  lib/psulib_traject.rb

Reply:
These have all been moved and namespaced in PsulibTraject module